packages/create-amplify/src/amplify_project_creator.ts

```diff
@@ -59,21 +54,12 @@ export class AmplifyProjectCreator {
     await logger.indicateProgress(`Creating template files`, async () => {
       await this.gitIgnoreInitializer.ensureInitialized();
 
-      await this.initialProjectFileGenerator.generateInitialProjectFiles();
+      await this.packageManagerController.generateInitialProjectFiles();
     });
 
     logger.log('Successfully created a new project!');
 
-    const cdCommand =
-      process.cwd() === this.projectRoot
-        ? ''
-        : `cd .${this.projectRoot.replace(process.cwd(), '')}; `;
-
-    logger.log(
-      `Welcome to AWS Amplify! 
-Run \`npx amplify help\` for a list of available commands. 
-Get started by running \`${cdCommand}npx amplify sandbox\`.`
-    );
+    logger.log(this.packageManagerController.getWelcomeMessage());
```

Comment:
What I meant by my other comment was leave the "cd" part of the welcome message here and move the package manager specific part to the controller. It'll require a little bit of modification to the message to make it work cleanly. Something like
```ts
const cdPreamble = process.cwd() === this.projectRoot
  ? ''
  : `Navigate to your project directory using
'cd .${this.projectRoot.replace(process.cwd(), '')}'.
Then get started with the following commands:
`;

logger.log(
`Welcome to AWS Amplify!
${cdPreamble}
${this.packageManagerController.getWelcomeMessage()}
`)
```
Then `getWelcomeMessage` is just the getting started commands part:
```ts
getWelcomeMessage: () => `
Run '${this.binaryRunner} amplify help' for a list of available commands.
Get started by running '${this.binaryRunner} amplify sandbox'.
`
```

Reply:
Done 3667fd2b53